heaps/heap_sort.py

```diff
@@ -1,8 +1,19 @@
 
-
+from heaps.min_heap import MinHeap
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(n log n)
+        Space Complexity: O(1)
```

Comment:
👍 However the space complexity is O(n) because you are building up a heap of `n` nodes as the function runs.